### PR TITLE
[MM-29183] - fix(post_info): add id prop to ToolTip

### PR DESCRIPTION
**Pull request by Phizzard**
## Summary

Fixes console error raising accessibility concerns with React-Bootstrap's <ToolTip /> component missing a value from it's Id prop.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/15690

### CODE REVIEW

File: components/post_view/post_info/post_info.jsx [hunk 1]
```diff
@@ -307,7 +307,7 @@ export default class PostInfo extends React.PureComponent {
                     delayShow={Constants.OVERLAY_TIME_DELAY}
                     placement='top'
                     overlay={
-                        <Tooltip>
+                        <Tooltip id='additional-information-overlay'>
```
Comment (hanzei) [inline components/post_view/post_info/post_info.jsx:310]: What happens if there are multiple posts with a tooltip?

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the contribution:+1: One question below

File: components/post_view/post_info/post_info.jsx (hunk 1 above)
Comment (Phizzard) [reply components/post_view/post_info/post_info.jsx:310]: Good question! I believe the `id` prop does get used as an `id` HTML attribute, which needs to be unique.

Should we add a new required prop to the `PostInfo` component for this value?

Comment (hanzei) [reply components/post_view/post_info/post_info.jsx:310]: Could we generate a unique id using e.g. the post id?

**Review by iomodo — APPROVED** [review]
LGTM! Thanks for the contribution @Phizzard

**Review by hanzei — APPROVED** [review]

File: components/post_view/post_info/post_info.jsx [hunk 2]
```diff
@@ -307,7 +307,7 @@ export default class PostInfo extends React.PureComponent {
                     delayShow={Constants.OVERLAY_TIME_DELAY}
                     placement='top'
                     overlay={
-                        <Tooltip>
+                        <Tooltip id={`additional-information-overlay-${post.id}`}>
```
Comment (saturninoabril) [inline components/post_view/post_info/post_info.jsx:310]: Could we change to
```suggestion
                        <Tooltip id={`viewAdditionalInfo-${post.id}`}>
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @Phizzard! Left a comment and I'm all good.

**Review by saturninoabril — APPROVED** [review]
Thanks @Phizzard for updating. LTGM

**Pull request merged**